tests/test_docker_latex.py

```diff
@@ -0,0 +1,51 @@
+import os
+import shutil
+import unittest
+import logging
+
+import quizgen.latex
+import quizgen.pdf
+import quizgen.util.dirent
+import quizgen.util.json
+import tests.base
+
+class TestDockerPdfConversion(tests.base.BaseTest):
```

Comment:
Most of this test is not docker-specific.
You should generalize this test class just for pdf conversion.
Put most of what you have here into a helper method,
and make a couple tests methods for docker and non-docker compilation.